coffee-electron/src/CoffeeForm.js

```diff
@@ -3,25 +3,59 @@ import { Form, Input } from 'semantic-ui-react';
 import './CoffeeForm.css';
 
 export const CoffeeForm = () => {
-  const [frequency, setFrequency] = useState('60');
+  const [secondsFrequency, setSeconds] = useState('0');
+  const [minutesFrequency, setMinutes] = useState('5');
```

Comment:
Based on React `useState` [convention](https://reactjs.org/docs/hooks-state.html#declaring-a-state-variable), the name of the state should match with the name of the setter, so:

```js
const [secondsFrequency, setSecondsFrequency] = useState('0');
const [minutesFrequency, setMinutesFrequency] = useState('5');
```